Rules/CommonScripts/GameStateBanners.as

```diff
@@ -0,0 +1,197 @@
+#define CLIENT_ONLY
+
+Vec2f bannerStart;
+Vec2f bannerPos;
+Vec2f bannerDest;
+f32 frameTime = 0.0f;
+const f32 maxTime = 0.6f;
+
+bool minimap = true;
+
+const u32 winBannerDuration = 8 * getTicksASecond();
+const u32 buildBannerDuration = 5 * getTicksASecond();
+const u32 gameBannerDuration = 5 * getTicksASecond();
+
+namespace Banner
```

Comment:
turn this into a normal enum and make the names reflect the game states (like the enum in `CTF_Banners` I suggested to be removed)

Reply:
resolved